code/game/objects/items/storage/backpack.dm

```diff
@@ -375,12 +375,17 @@
 
 /obj/item/storage/backpack/marine/satchel
 	name = "\improper TGMC satchel"
-	desc = "A heavy-duty satchel carried by some TGMC soldiers and support personnel."
+	desc = "A heavy-duty satchel carried by some TGMC soldiers and support personnel. This one is in black."
```

Comment:
The descriptions do not need color information, people will see that just by looking at the sprite.